### DISCUSSION TITLE
Is it possible to list all user Ids from a specific guild using the console?

### DISCUSSION THREAD

**Discussion by CloudTheFoxdev** [rigwild/discord-self-bot-console#64] [Q&A]
Yeah, I am looking for a way to list all user Ids from the current guild I am in. I know you can do so with discord.js but I get errors when it comes to the client console. I'm still a beginer and its the only repository I've seen about the client console and its good stuff thank you.

I'm french sorry for bad spelling.

**Comment by DeadLyBro** [comment]
I tried [this](https://discord.com/developers/docs/resources/guild#list-guild-members) and I got error: 50001 Missing Access.
